simple_api/django_object/validators.py

```diff
@@ -0,0 +1,29 @@
+from django.core.exceptions import ValidationError
+from simple_api.object.validators import Validator
+
+
+class DjangoValidator(Validator):
+    def __init__(self, field_name, validation_fn):
+        self.fn = validation_fn
+        self.field_name = field_name
+
+    def validation_statement(self, request, value=None, **kwargs):
+        try:
+            self.fn(value)
+            return True
+        except ValidationError:
+            return False
+
+    def error_message(self, **kwargs):
+        return "Django validator failed in field {}".format(self.field_name)
+
+
+class ForeignKeyValidator(Validator):
```

Comment:
zatim asi OK takhle, podle me se casem ukaze, ze je to prilis coarse-grained

Reply:
Possible, uvidíme